### PR TITLE
refactor: fixed links to use new org

### PR DESCRIPTION
**Pull request by Fifciu**
### Short Description and Why It's Useful

As we changed the organization of the repository - I also changed links to use new org name.

### Which Environment This Relates To
Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)

- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
- [x] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`

### Upgrade Notes and Changelog
- [ ] No upgrade steps required (100% backward compatibility and no breaking changes)
- [x] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature

**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/DivanteLtd/vue-storefront/blob/master/CONTRIBUTING.md)

### CODE REVIEW

**Comment by Fifciu** [comment]
https://github.com/divanteltd/vsf-default needs to be moved to our organization

File: .github/workflows/test_vsf1.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ jobs:
           key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
 
       - name: Clone default theme
-        run: git clone --quiet --single-branch --branch master https://github.com/DivanteLtd/vsf-default.git ./src/themes/default
+        run: git clone --quiet --single-branch --branch master https://github.com/vuestorefront/vsf-default.git ./src/themes/default
```
Comment (gibkigonzo) [inline .github/workflows/test_vsf1.yml:37]: this pr should be merged **after** `vsf-default` is moved to `vuestorefront`

File: core/i18n/resource/i18n/ar-SA.csv [hunk 2]
```diff
@@ -1,77 +1,77 @@
-Registering the account ...,جاري تسجيل الحساب
-No products synchronized for this category. Please come back while online!,.لا يوجد منتجات تمت مزامنتها لهذا التصنيف. يرجى العودة مرة أخرى إذا تم الاتصال بالإنترنت
-Shopping cart is empty. Please add some products before entering Checkout,سلة التسوق فارغة. يرجى إضافة بعض المنتجات للانتقال إلى صفحة إتمام الطلب
-Out of stock!,غير متوفر!
  is out of the stock!,غير متوفر بالمخزون!
```
Comment (gibkigonzo) [inline core/i18n/resource/i18n/ar-SA.csv:1]: Does it work at all? it should be `"is out of the stock!","غير متوفر بالمخزون!"`

File: .github/workflows/test_vsf1.yml (hunk 1 above)
Comment (filrak) [reply .github/workflows/test_vsf1.yml:37]: done, thx ;)

**Review by gibkigonzo — APPROVED** [review]

File: core/i18n/resource/i18n/ar-SA.csv (hunk 2 above)
Comment (Fifciu) [reply core/i18n/resource/i18n/ar-SA.csv:1]: Looks like it works fine

**Pull request merged**